### ISSUE TITLE
Add reducer chaining method to all action:

### ISSUE THREAD

**Issue by alfnielsen** [alfnielsen/redux-area#5]
You can at the moment call: (Chain = call logic from other reducer)
```ts
.success((draft, { values }) => {
    OtherAction.reducer(draft, {     
      type: OtherAction.name, 
      ...values 
    )
})
```
So it could be nice to be able to chain more fluent like:

```ts
.success((draft, payload) => {
    OtherAction.apply(draft, { ...values })
})
```

**Comment by alfnielsen** [comment]
Actually you can't even do the first thing, but the this functionality could be good!

**Comment by alfnielsen** [comment]
There are now an experimental version of this, called `use`
```ts
OtherAction.use(draft, { type: '', ...payload})
```
Will try to remove the required unused 'type' before the final version

**Comment by alfnielsen** [comment]
Implemented in version 0.2.0
